packages/workflow/src/Interfaces.ts

```diff
@@ -35,7 +35,7 @@ export type IAllExecuteFunctions =
 	| ITriggerFunctions
 	| IWebhookFunctions;
 
-export type BinaryFileType = 'text' | 'json' | 'image' | 'video';
+export type BinaryFileType = 'text' | 'json' | 'image' | 'audio' | 'video' | 'pdf';
```

Comment:
should we use `document` instead of `pdf` here? are there any other document types that browsers can easily render via the `<embed>` tag, that are also relevant to us?

Reply:
I can see more documents being needed.. but as long as users can download them, it's fine as is. or until a user requests it